src/project/DownloadFlashButton.tsx

```diff
@@ -86,6 +88,7 @@ const DownloadFlashButton = ({ size }: DownloadFlashButtonProps) => {
                 </MenuItem>
               )}
               <MenuItem
+                as="a"
                 target="_blank"
```

Comment:
These are the download/flash buttons so these shouldn't be links at all. They also lose the button cursor icon with this change. Remove `as`, `target` and `rel` props from all?

Reply:
Oops, yes, thank you.

Reply:
[caddd1f](https://github.com/microbit-foundation/python-editor-next/pull/564/commits/caddd1f795951080bb08229c1c2b10246d0c5dcb)

Reply:
... and ca8e892cfd62b3b84ed6e2376cb97beedbb52f2c